frontend/src/elements/components/Consent.js

```diff
@@ -72,6 +72,10 @@ class Consent extends React.PureComponent {
     let data = {};
     let processors = this.state.processors.slice();
 
+    if (processors.length < 1 && !ReactDOM.findDOMNode(this)) {
```

Comment:
Why is that?
We should be able to consent to only the controller.

Reply:
Hm, that's an interesting question then, I will modify to allow that, makes sense.